apps/leo_gateway/priv/leo_gateway.conf

```diff
@@ -109,6 +109,16 @@ bucket_prop_sync_interval = 300
 ## wtperf is set to the same value.
 ## nfs.nfsd.wtmax = 5242880
 
+## NFSd's Readdir Entry Table Scan Interval in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_scan_int = 1800
+
+## NFSd's Readdir Entry TTL in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_entry_ttl = 3600
+
+## NFSd's Readdir Entry Table Memory Scanning Threshold in bytes (leo-project/leofs#612)
+## nfs.nfsd.readdir_mem_thres = 268435456
+
```

Comment:
@windkit As I said at the previous comment,
Please use not the `dummy` as a value in tests but the actual example leo_nfs_proto3_server has used for storing a result of leo_nfs_file_handler:list_dir.
I'd like to decide the ideal default based on the actual data as I'm not sure how much the current default(256MB) can store.